seacrowd/sea_datasets/vitext2sql/vitext2sql.py

```diff
@@ -0,0 +1,170 @@
+            # in the meantime just treat it as large string.
+            features = datasets.Features(
+                {
+                    "db_id": datasets.Value("string"),
+                    "query": datasets.Value("string"),
+                    "query_toks": [datasets.Value("string")],
+                    "query_toks_no_value": [datasets.Value("string")],
+                    "question": datasets.Value("string"),
+                    "question_toks": [datasets.Value("string")],
+                    "sql": datasets.Value("large_string"),
+                }
+            )
+        elif self.config.schema == "seacrowd_t2t":
+            features = schemas.text2text_features
+
+        return datasets.DatasetInfo(
+            description=_DESCRIPTION,
+            features=features,
+            homepage=_HOMEPAGE,
+            license=_LICENSE,
+            citation=_CITATION,
+        )
+
+    def _split_generators(self, dl_manager: datasets.DownloadManager) -> List[datasets.SplitGenerator]:
+        urls = _URLS[_DATASETNAME]
+
+        if "syllable" in self.config.name:
+            level_urls = urls["syllable-level"]
+        else:
+            level_urls = urls["word-level"]
+
+        data_files = {
+            "train": dl_manager.download_and_extract(level_urls["train"]),
+            "test": dl_manager.download_and_extract(level_urls["test"]),
+            "validation": dl_manager.download_and_extract(level_urls["validation"]),
+        }
+        split_generators = [
+            datasets.SplitGenerator(
+                name=datasets.Split.TEST,
+                gen_kwargs={
+                    "filepath": data_files["test"],
+                },
+            ),
+            datasets.SplitGenerator(
+                name=datasets.Split.TRAIN,
+                gen_kwargs={
+                    "filepath": data_files["train"],
+                },
+            ),
+            datasets.SplitGenerator(
+                name=datasets.Split.VALIDATION,
+                gen_kwargs={
+                    "filepath": data_files["validation"],
+                },
+            ),
+        ]
+
+        return split_generators
+
+    def _generate_examples(self, filepath: Path) -> Tuple[int, Dict]:
+        df = pd.read_json(filepath)
+        if self.config.schema == "source":
+            for i, row in df.iterrows():
+                entry = {"db_id": row["db_id"], "query": row["query"], "query_toks": row["query_toks"], "query_toks_no_value": row["query_toks_no_value"], "question": row["question"], "question_toks": row["question_toks"], "sql": str(row["sql"])}
+                yield i, entry
+        elif self.config.schema == "seacrowd_t2t":
+            for i, row in df.iterrows():
+                entry = {
+                    "id": str(i),
+                    "text_1": row["question"],
+                    "text_2": row["query"],
+                    "text_1_name": "question",
+                    "text_2_name": "query",
```

Comment:
`Query` can be misleading, change into `sql_query` 
```suggestion
                    "text_1_name": "question",
                    "text_2_name": "sql_query",
```

Reply:
did the revisions!